styles/components/ItemTable.module.scss

```diff
@@ -9,23 +9,39 @@
 
   tr {
     th,
-    td,
+    td {
+      padding: null var(--nypl-space-xs);
+    }
+
+    th {
+      text-transform: uppercase;
+    }
+
     td:first-of-type,
     th:first-of-type {
-      text-transform: none;
-      padding: var(--nypl-space-s) 0;
+      padding: var(--nypl-space-s) var(--nypl-space-xs) var(--nypl-space-s) 0;
+    }
 
+    td:last-of-type,
+    th:last-of-type {
+      padding: var(--nypl-space-s) 0 var(--nypl-space-s) var(--nypl-space-xs);
+    }
+
+    th,
+    td,
+    td:first-of-type,
+    th:first-of-type {
       // These min-widths are here temporarily to fix an issue with the table that causes the Format and Call number columns to appear to narrow
```

Comment:
Address the TODO and the comment above. The `Table` update allows a way to update the column width in a more standard way.